src/microbe_stage/systems/MicrobePhysicsCreationAndSizeSystem.cs

```diff
@@ -499,6 +499,12 @@ private void UpdateNonPhysicsSizeData(in Entity entity, float membraneRadius, re
             entity.Get<CompoundAbsorber>().AbsorbRadius =
                 Math.Max(cellProperties.Radius, Constants.MICROBE_MIN_ABSORB_RADIUS);
         }
+
+        if (entity.Has<CurrentAffected>())
+        {
+            entity.Get<CurrentAffected>().EffectStrength = cellProperties.Radius
```

Comment:
I think that this system will need to be marked as writing to the `CurrentAffected` component as I can't recall that this system would already be marked as touching that.